ui/src/dashboards/components/Dashboard.js

```diff
@@ -17,11 +20,14 @@ const Dashboard = ({
   inPresentationMode,
   onOpenTemplateManager,
   onSummonOverlayTechnologies,
+  onSelectTV,
```

Comment:
if the API wants a key called `templates`, then maybe everything should be `templates` rather than `tempVars`? for me, `templates` feels not descriptive enough, but better to be consistent.